locales/ku_KU/duckduckgo.po

```diff
@@ -0,0 +1,5940 @@
+#
+# Autogenerated by DDGC::LocaleDist 0.0development
+#
+# language: Kurdish
+# locale: ku_KU
+# date: 2019-01-16 23:43-0500
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: DuckDuckGo-Translation-0.000\n"
+"POT-Creation-Date: 2019-01-16 23:42-0500\n"
+"PO-Revision-Date: 2019-01-16 23:43-0500\n"
+"Last-Translator: Community\n"
+"Language-Team: DuckDuckGo Community <community@duckduckgo.com>\n"
+"Language: Kurdish (Kurdistan)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+msgid "!Bang Search Shortcuts"
+msgstr "!Bang kurteyên lêgerînê"
+
+#. Heading for the table column indicating what percent of the population is fully vaccinated
+msgctxt "Covid 19 module"
+msgid "% Fully Vaccinated"
+msgstr "% 100% Derzî kirî"
+
+#. Heading for the table column indicating what percent of the population is partially vaccinated
+msgctxt "Covid 19 module"
+msgid "% Vaccinated"
+msgstr "% Derzî kirî"
+
+msgctxt "SERP footer content"
+msgid "%s Billion Searches"
+msgstr "%s Milyar Lêgerîn"
+
+#. Used to indicate the season year, e.g. '2020 Season'
+msgctxt "Sports module"
+msgid "%s Season"
+msgstr "%s Demsal"
+
+msgid "%s blocked by safe search."
+msgstr "%s ji aliya lêgerîna parastî ve hate qedexe kirin."
+
+#. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+msgctxt "published dates for organic results"
+msgid "%s days ago"
+msgstr "%s roj berê"
+
+#. An indication that inputted text was detected to be in this language. E.g.: Italian detected.
+msgctxt "translations_module"
+msgid "%s detected"
+msgstr "%s hate dîtin"
+
+#. in https://duckduckgo.com/params. multiple fors
+msgid "%s for %s"
+msgstr "%s ji bo %s"
+
+#. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+msgctxt "published dates for organic results"
+msgid "%s hours ago"
+msgstr "%s demjimêr berê"
+
+msgctxt "showcase_donations"
+msgid "%s in privacy donations!"
+msgstr "%s Bexşandina di taybetî de!"
+
+#. Places distance in km
+msgid "%s km"
+msgstr "%s km"
+
+#. Places distance in miles
+msgid "%s mi"
+msgstr "%s mi"
+
+#. This is used to add attribution to user reviews on websites, e.g. "John Doe on Yelp"
+msgctxt "maps_places"
+msgid "%s on %s"
+msgstr "%s li ser %s"
+
+msgctxt "install-duckduckgo"
+msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
+msgstr "%sMafê bide %sKontrola%sTevlîkirinê%s, paşê %sBaş e%s Bitikîne"
+
+msgctxt "install-duckduckgo"
+msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
+msgstr "%sMafê bide %sKontrola%sTevlîkirinê%s, paşê %sBaş e%s Bitikîne"
```

Comment:
In both of these strings the translations don't contain the same number of placeholders than the source. It seems that the two translations are the same, but the source strings are different.